### DISCUSSION TITLE
New-HTMLETable -SearchPane

### DISCUSSION THREAD

**Discussion by TrondSO** [EvotecIT/PSWriteHTML#412] [Ideas]
Hi, hope you still are updating this module.  Just started to use your module, and it's great and fun :)

Have something that would be awsome if you could implement. When using New-HTMLTable -SearchPane, that you can name what colom should be in the SearchPane. If you have only 4 colums that you would like to search on but ending up with 10 it's abit messy on top there :). Just a thought that would be great.

(Did not see my post, so hope i'm dobble posting..)

**Comment by PrzemyslawKlys** [comment]
There are many configurations available in SearchPanes, but adding them to PSWriteHTML is expensive.
- https://datatables.net/extensions/searchpanes/examples/

But if I will have time, or someone else will - it's open. What you want is I guess this:
- https://datatables.net/extensions/searchpanes/examples/initialisation/layout.html

**Reply by TrondSO** [reply]
Tnx for answer, I'm more like a novice in ps still :). So until or if you find a solution to this. I will try to filter the data I get from SQL and make more page's :) . Yea the example is like whant I want to do. Getting like 10 collums and only have SearchPanes for 4 of the collums. Or else thoes SearchPanes will take alot of space on top :).

Keep up the good work, love the module :).

**Reply by PrzemyslawKlys** [reply]
You can always use SearchBuilder which is enabled by default

**Reply by TrondSO** [reply]
hmm, true. Sry, did not think of that :) Tnx will try that one out :).
